src/EnergyPlus/SolarShading.cc

```diff
@@ -6849,6 +6828,8 @@ namespace SolarShading {
 
             // Loop over exterior surfaces in this zone
             auto &thisEnclosure(DataViewFactorInformation::ZoneSolarInfo(enclosureNum));
+            // delete values from previous timestep
+            if (AnyBSDF) AWinCFOverlap = 0.0;
```

Comment:
Fix #8088. Maybe should remove that from here and make a separate PR for this, but don't want to wait for that anymore.